### ISSUE TITLE
User-facing document about Locations and Placements

### ISSUE THREAD

**Issue by davidfestal** [kcp-dev/kcp#1913]
We should provide user-facing document about Locations and Placement that:
- Briefly explain the personas involved in the overall flow
- Focus only on the concepts useful for the user of the KCP workspace who wants to have his workloads schedule on some available locations.

Note: this might be worth waiting for the full Location Workspace story to be implemented before writing this doc.

**Comment by ncdc** [comment]
I'm guessing this is different than what's already in locations-and-scheduling.md?

**Comment by davidfestal** [comment]
> I'm guessing this is different than what's already in locations-and-scheduling.md?

Yes, the idea is to provide only the required level of information for a user to have its workload scheduled and running, without necessarily understanding "how it all works"

**Comment by stevekuznetsov** [comment]
> this might be worth waiting for 

I'd argue that whatever is implemented, is documented. If and only if there is no user-facing flow at all, do we not have user-facing docs.

**Comment by cmoulliard** [comment]
The overall idea around this documentation improvement is to explain what the different users (Developer, Project or Architect Managers, Cloud Infrastructure engineer) using kcp are supposed to do ? 

Example: Is a developer supposed to create a placement ? If yes, what should then the placement includes as information (I want to access a cluster, I want to access a namespace supporting my CRDs or k8s resources, .... ) ? Is the purpose of a placement to claim a workspace in a k8s cloud infrastructure matching my needs ?
